server/events/templates/multi_project_import.tmpl

```diff
@@ -1,3 +1,10 @@
 {{ define "multiProjectImport" -}}
-{{ template "multiProjectApply" . }}
-{{- end }}
+{{ template "multiProjectHeader" . }}
```

Comment:
This template is very different too

Reply:
based on https://github.com/runatlantis/atlantis/pull/2880#discussion_r1057330962, `import` template should not use `multiProjectApply` template. So expand it.